pkg/test/nodepool.go

```diff
@@ -51,6 +51,9 @@ func NodePool(overrides ...v1beta1.NodePool) *v1beta1.NodePool {
 	if override.Spec.Template.Spec.Requirements == nil {
 		override.Spec.Template.Spec.Requirements = []v1beta1.NodeSelectorRequirementWithMinValues{}
 	}
+	if override.Status.Conditions == nil {
```

Comment:
Is there a reason we would not want to set the status conditions in this way in the helper? I guess we can always force it to false later if we need?